zzzz_modular_occulus/code/game/gamemodes/occultist/occultist_machines.dm

```diff
@@ -0,0 +1,274 @@
+	var/partsamount
+	var/list/blacklist = list(/material/resin, /material/voxalloy, /material/plastic/holographic, /material/wood/holographic)//A blacklist of materials we don't want spawning, minus the padding because that's handled on deconstruction
+	var/list/partslist //A list of parts for random choosing when it's time to deconstruct.
+
+/obj/machinery/occultist/totem/New(var/loc, var/cultist)
+	maker = cultist //Set the owner for remote madness generation
+	//Pick a name!
+	name = pick(\
+		"\proper the unsettling question",
+		"\proper the unbound sky",
+		"\proper the thing in the corner of your eye",
+		"\proper the shadow standing behind you",
+		"\proper the smallest infinity",
+		"\proper the greatest nothingness",
+		"\proper the blood-blister upon spacetime",
+		"\proper the melody of a blue space",
+		"\proper the scent of water on smoke"\
+		)
+
+	//The next thing we need to do is pick the tool quality that can deconstruct it.
+	deconqual = pick(deconlist)
+	if(deconqual == QUALITY_BOLT_TURNING)
+		descdecon = "It looks like it's haphazardly bolted together."
+	if(deconqual == QUALITY_CUTTING)
+		descdecon = "It's bound together by some thick cord."
+	if(deconqual == QUALITY_DRILLING)
+		descdecon = "You might need to drill the joins apart."
+	if(deconqual == QUALITY_HAMMERING)
+		descdecon = "The joins look particularly weak. Smashing them might help."
+	if(deconqual == QUALITY_PRYING)
+		descdecon = "A prybar might be able to pop this apart."
+	if(deconqual == QUALITY_PULSING)
+		descdecon = "It's held together by some electronic mechanism."
+	if(deconqual == QUALITY_SAWING)
+		descdecon = "The frame looks thin. A saw might help."
+	if(deconqual == QUALITY_SCREW_DRIVING)
+		descdecon = "There are very obvious screws holding the whole thing together."
+	if(deconqual == QUALITY_WELDING)
+		descdecon = "It's poorly welded together."
+	if(deconqual == QUALITY_WIRE_CUTTING)
+		descdecon = "Wire holds the structure together."
+
+	//Pick a lore blurb for the desc. The rest of the desc will be settled later.
+	desclore = pick(\
+		"Are you the real you? Or are you someone else?",
+		"Sometimes the mind is too free.",
+		"This wasn't here a second ago.",
+		"It shimmers and contorts unnaturally.",
+		"How do you cure someone of their madness without curing them of their identity?",
+		"The first step towards freedom is death.",
+		"Your bones hum in harmony with the rumbling of the ship. Let them out.",
+		"It moves when it thinks you aren't looking at it.",
+		"There is a hole in your head where your mind should be, but you don't feel any different.",
+		"Is belief the key? It- no, it's too late for it to matter."\
+		)
+	//Now we slap the description together. First the lore then how to take it apart.
+	desc = "[desclore] [descdecon]"
+
+//Set the look. Put these in zzzz_modular_occulus/icons/turf/occultist/misc.dmi
+	icon_state = pick("totem[rand(1,6)]")
```

Comment:
Do note, that the `totem` icon_state will have to be changed to `totem1`, if it wasn't already.

Reply:
Fixed.